### DISCUSSION TITLE
CSS Not Working

### DISCUSSION THREAD

**Discussion by vishwizard** [vercel/community#7302] [Help]
### Summary

I created a project usinng Next.js.
here's the github link - https://github.com/vishwizard/nextecom
project deployment link - nextecom.digilopment.in

### Example

https://www.github.com/vishiwzard/nextecom

### Steps to Reproduce

Go to my repository at - https://www.github.com/vishwizard/nextecom
clone repo
add .env file
set up variables 
GOOGLE_ID="anytestoauthid"
GOOGLE_SECRET="oauthsecret"
MONGODB_URI="mongodburi"
NEXTAUTH_SECRET="anyrandomstring"

Run npm i to install dependencies
npm run dev to run in dev mode
npm run build to run in production

The build mode too works fine in my localhost but when I deploy to vercel, The Css breaks

**Comment by pawlean** [comment]
Hey @vishwizard!

A similar discussion was raised previously, can you check the answers shared [here](https://github.com/orgs/vercel/discussions/5063)?

**Reply by vishwizard** [reply]
Hey @pawlean,
Thanks for a response,
I went through it but it still does not resolve my issue
The problem is it's a next js project and the folder structure of mine is totally different from theirs. 
simply renaming the css file doesn't work either, I tried.

[a comment by an automated account is not shown]
